src/renderer/components/HeadlessScenePlayer.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
```

Comment:
This file was factored out of `Player.tsx`.

Before, we had one Player component that was responsible for loading images and controlling playback, including history, for one scene.

Now we have a separation of playback controls (still in Player) from the loading and display of a specific scene (HeadlessScenePlayer). The naming could probably be improved but the separation of concerns is definitely better.